client/components/TestRun/index.jsx

```diff
@@ -1390,9 +1390,24 @@ const TestRun = () => {
                         isAdmin={isAdminReviewer}
                         atName={testPlanReport.at.name}
                         atVersion={currentTest.testResult?.atVersion?.name}
-                        atVersions={testPlanReport.at.atVersions.map(
-                            item => item.name
-                        )}
+                        atVersions={testPlanReport.at.atVersions
+                            .filter(item => {
+                                // Only provide at version options that released
+                                // at the same time or later than the minimum
+                                // AT version
+                                let earliestReleasedAt = null;
+                                if (testPlanReport.minimumAtVersion) {
+                                    earliestReleasedAt = new Date(
+                                        testPlanReport.minimumAtVersion.releasedAt
+                                    );
+                                    return (
+                                        new Date(item.releasedAt) >=
+                                        earliestReleasedAt
```

Comment:
At some point it might be good to break this out into a utility function so that we can unit test these comparisons to harden against any future changes to ATVersion. Not a requirement right now and I can easily add when doing the automation at version comparison work if preferred.

Reply:
I think I'll leave it up to a future PR because since there's a couple places where some date comparison is being done. Would rather some dedicated time be taken to pull those all together if creating a utility on that.

> ... and I can easily add when doing the automation at version comparison work if preferred

@stalgiag that's my preference right now, but great call out on this and definitely include me as a reviewer when doing so!